app.py

```diff
@@ -191,9 +191,10 @@ def handle_delete_education():
     Handle education delete requests
     '''
     index = request.args.get("index")
-    if index is not None:
-        return delete_education_by_index(data, index)
-    return jsonify(data["education"])
+    existing_education = get_education_by_index(data, index)
```

Comment:
Unclear what's changed. If there's an error, it still returns the full list?

Reply:
if SERVER_ERROR in existing_education.json:
        return existing_education
    return delete_education_by_index(data, index)
    
These are the changes I made and pushed them